docs/server-side/v2/changelog/release-notes.mdx

```diff
@@ -5,6 +5,10 @@ nav: 5.1
 
 This Changelog highlights notable changes to the 100ms server-side API, such as API additions, improvements, and deprecations. Also, we've included developer experience improvements to this page to keep you on track with items that will enhance your integration journey.
 
+### 2023-10-11
```

Comment:
Use ##

Reply:
Some older ones had ### as well. Fixed that. Thanks!